src/lib/snarky_bn382_backend/dlog_based.ml

```diff
@@ -141,7 +140,7 @@ module Keypair = struct
     in
     (set_urs_info, load)
 
-  let () = set_urs_info "/home/izzy/pickles-new/dlog-urs"
+  let () = set_urs_info "/home/pavel/urs/dlog-urs"
```

Comment:
We should delete this or maybe make a file in `/tmp` or something

Reply:
This has been addressed in commit 3e28ef7ec16d58af0d3c8ba087f5cad92d44907d